main.nf

```diff
@@ -21,78 +21,66 @@ version = '0.1'
 
 // Configurable variables - same as NGI-RNAseq for now
 params.project = false
-params.genome = false
-params.forward_stranded = false
-params.reverse_stranded = false
-params.unstranded = false
-params.star_index = params.genome ? params.genomes[ params.genome ].star ?: false : false
-params.fasta = params.genome ? params.genomes[ params.genome ].fasta ?: false : false
-params.gtf = params.genome ? params.genomes[ params.genome ].gtf ?: false : false
-params.bed12 = params.genome ? params.genomes[ params.genome ].bed12 ?: false : false
-params.hisat2_index = params.genome ? params.genomes[ params.genome ].hisat2 ?: false : false
-params.multiqc_config = "$baseDir/conf/multiqc_config.yaml"
-params.splicesites = false
-params.download_hisat2index = false
```

Comment:
I prefer to delete stuff like this unless we're sure that it's going to be used soon. Can always add it back easily enough.